test/utility/misc.c

```diff
@@ -139,6 +139,20 @@ testutil_cleanup(TEST_OPTS *opts)
 }
 
 /*
+ * testutil_check_stop_long --
+ *	Check if WT_DISABLE_LONG_TESTS is set and exit if so
```

Comment:
It would be better if the environment variable and function names match, since they have the same structure. How about making them ``testutil_disable_long_tests`` and ``TESTUTIL_DISABLE_LONG_TESTS``